feat: add in conference details (#90)
uk-x-gov-software-community/uk-x-gov-software-community.github.io: conference-2025-01-29.md

On Wednesday, 29th January 2025, we will be hosting an in-person conference in London, bringing together members of the community for an exciting day of talks, discussions, and networking. 

<a href="#the-agenda-of-the-event" title="agenda" >Agenda of the event</a>
<a href="#the-venue" title="venue" >The venue </a>
<a href ="#how-to-get-there" title= "How to get there" >How to get there </a>
<a href="#on-the-day" title="On the day"> On the day </a>
<a href= "#if-you-are-staying-overnight" title="If you are staying overnight">If you are staying overnight</a>

### The Agenda of the event 
The Cross Government Software Engineering Conference will take place on Wednesday, 29th January 2025 from 9 AM to 5PM, social (optional) from 5PM onwards. Below the breakdown of the sessions:

| Time |  Main Room | Breakout room 1 |
|-----|---|---|
| 09:00 - 10:30 | Registration | |
| 10:30 - 11:30 |  Keynote - Asmin Hussian, Green Software Foundation (with Q&A)
| 11:30 - 13:00 | Lightning Talks (6 sessions with Q&A) | Lean Coffee Session |
| 13:00 - 14:00 | Lunch ||
| 14:00 - 15:00 |  Keynote - Terence Eden, Lessons learned Open Sourcing the UK's Covid Tracing App ||

> | 15:00 - 16:30 |  Deep-dive tech sessions | Lean Coffee Session |

Reviewer: Reflecting Claudia's suggestion
```suggestion
| 15:00 - 15:20 |  Tech Talk TBC | Lean Coffee Session |
```

Join us after the conference at The Running Horse pub from 5:00 PM onwards. The venue offers a wide selection of craft beers and food options catering to all dietary requirements.

### The Venue
The event will take place at the Whitechapel building, 10 Whitechapel High St, London E1 8QS.
The event will be hosted at the GDS offices, on the 7th floor of the building.
1 week before the event you will receive an email with a day pass to access the building: you will need to activate your day pass at the ground floor reception by showing your ID and this email. You will need to keep the pass with you all day to enter and leave the venue.
You will also need to request a visitor pass at the office reception on the 7th floor: please bring your Civil Service pass with you to receive an unescorted visitor pass.
Don't worry: organisers will be available on the floors to direct you and help you get access!

### How to get there
The venue is located in Zone 1, and efficiently connected by several public transport lines. The nearest stations are:
* Liverpool Street (west) - 7-minute walk
* Aldgate and Aldgate East - 1-minute walk

N.B. There is no parking on site.

### On the day

> Please remember to bring a valid form of ID and, where applicable, a Civil Service pass.

Reviewer: ```suggestion
Please remember to bring a valid form of ID and preferably a Civil Service pass.
```

On the day you will also be given a conference pass to keep with you at all times, so that you can access the areas reserved for the event.
You will be able to use GovWifi on the premises.
Food and beverages, including tea and coffee, will be provided, please contact the organising committee if you have any specific request (secretariat-engineering-conference@digital.cabinet-office.gov.uk).
The venue also has cutlery, glasses and mugs for everyone to use: however, as we will use the office premises, please bring your own mug/flask to minimise disruption to colleagues working.

### If you are staying overnight
You can book a wide range of hotels in the area through CTM or any other provider (just make sure  they are within the budget policy for London). However we would sugesst the following close by hotels:

* Holiday Inn Whitechapel
* Hub by Premier Inn London Spitalfields
* ibis London City - Shoreditch
* Wilde apartments, Commercial street

> * Leman Locke studios, Leman street

Reviewer: ```suggestion
* Leman Locke studios, Leman street
* Travelodge London Central Aldgate East
```

Just as CTM suggested this also for me!
